src/class.c

```diff
@@ -1023,8 +1023,10 @@ mrb_get_args(mrb_state *mrb, const char *format, ...)
   if (!c && argc > i) {
     mrb_raise(mrb, E_ARGUMENT_ERROR, "wrong number of arguments");
   }
+
+finish:
   va_end(ap);
-  return i;
+  return argc;
```

Comment:
Is there any reason to replace `i` with `argc` here? We checked `i == argc` before jumping to `finish`.

Reply:
That is my mistake. sorry.
I will fix it.